string[] ag_name  			# agent name (label) - sender
string ag_addr  			# agent address (namespace) - sender
string mission_sign
string[] task_labels
string[] neighbors
